app/pendrive_detection/pendrive_detector.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+from typing import Optional
+
+if sys.platform == "linux":
+    import pyudev
+    import psutil
+
+if sys.platform == "win32":
+    import wmi
+
+
+class PenDriveFinder:
+    """
+    A utility class to detect pen drives and locate private key files (*.pem) stored
+    on them on various operating systems(Windows, Linux, macOS).
+    """
+
+    def __init__(self):
+        self._wmi_client = None
+
+    @staticmethod
+    def check_os() -> str:
```

Comment:
does it really necessary to have wrapper around sys.platform, because as I see there is no additional logic